src/test/java/com/hocs/test/glue/WorkstacksStepDefs.java

```diff
@@ -39,6 +40,77 @@
 
     private Page page; //more pages
 
+    Homepage homepage;
+
+    Workstacks workstacks;
+
+    @When("^I allocate the case to myself$")
+    public void allocateToMe() {
+        homepage.firstStageFindMyCase();
+        workstacks.clickAllocateToMeButton();
+        homepage.goHome();
+
+    }
+
+    @When("^I unallocate the case from myself$")
+    public void unallocateCase() {
+        homepage.selectMyCases();
+        workstacks.clickCheckboxWhereCaseReferenceIs();
```

Comment:
Similarly here 'where caseReference is' could be `checkboxRelevantToCaseReference` or `onCaseReferenceRow` perhaps